playbooks/roles/airship-setup-deployer/tasks/kubectl-install.yml

```diff
@@ -1,96 +0,0 @@
----
-- name: Download latest stable kubectl
-  become: yes
-  shell: >-
-    kubectl version --client || (curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl &&
-    chmod 0755 ./kubectl &&
-    mv ./kubectl /usr/local/bin/kubectl &&
-    ln -s /usr/local/bin/kubectl /usr/bin/kubectl)
-  register: _kubectldl
-  args:
-    executable: /bin/bash
-    chdir: /root
-    warn: False
-  changed_when: _kubectldl.stdout.find('Client Version') == -1 # If no version found, means it is installing
-
-- name: Ensure kube folder exists
-  file:
-    state: directory
-    path: "{{ ansible_user_dir }}/.kube"
-    mode: "0700"
-
-- name: Check if automation tooling files are present
-  stat:
-    path: /opt/ccp/environment.json
-  register: automation_environment_json
-
-- name: Use automation tooling if possible
-  when: automation_environment_json.stat.exists
-  block:
-  - name: Find kubeconfig location
-    command: jq -r '.kubeConfig.admin' /opt/ccp/environment.json
-    changed_when: false
-    register: _kubeconfiglocation
-
-  - name: Ensure kubeconfig is recent
-    stat:
-      path: "{{ _kubeconfiglocation.stdout_lines[0] }}"
-    register: _kubeconfigfromautomation
-
-  #NOTE: This task needs to run under bash, and therefore the shell module
-  #(instead of command) has to be used. Therefore, skipping of linting
-  #as to be done.
-  - name: Re-run automation if file is old
-    when: ansible_date_time.epoch|float - _kubeconfigfromautomation.stat.mtime > 86400
-    shell: ./velum-interactions -k -e /opt/ccp/environment.json
-    args:
-      chdir: /opt/ccp/automation/velum-bootstrap
-      executable: /bin/bash
-    tags:
-      - skip_ansible_lint
-
-  - name: Copy kubeconfig file
-    copy:
-      remote_src: yes
-      src: "{{ _kubeconfiglocation.stdout_lines[0] }}"
-      dest: "{{ ansible_user_dir }}/.kube/config"
-      mode: "0600"
-
-- name: Else use files from runner
-  when: not automation_environment_json.stat.exists
-  block:
-    - name: Find if runner has a kubeconfig file
-      stat:
-        path: "{{ kubeconfig_file_path }}"
-      delegate_to: localhost
-      register: new_config
-
-    - name: Find if an existing .kube/config file
-      stat:
-        path: "{{ ansible_user_dir }}/.kube/config"
-      delegate_to: localhost
```

Comment:
playbooks/roles/deploy-osh/tasks/kubectl-install.yml does not have this delegation.

Reply:
Right, it was changed with https://github.com/SUSE-Cloud/socok8s/pull/259